src/workflows/minting.ts

```diff
@@ -21,7 +21,7 @@ export async function mintingWorkflow(
     ether_key: user.user,
     tokens: user.tokens.map(token => ({
       id: token.id,
-      ...(token.blueprint && { blueprint: token.blueprint }),
+      blueprint: token.blueprint,
```

Comment:
Why this change?

Reply:
generating api code brought change to `blueprint` property and it not optional anymore.
This code required fix corresponding to that change.